samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,202 @@
+---
+page_type: sample
+description: This example demonstrates how Azure Open AI extracts action items from meeting transcriptions for all participants who subscribed to the meeting, and subsequently sends these action items to each individual user in a 1:1 chat after the meeting concludes.
+products:
+- office-teams
+- copilot-m365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "07/29/2024 01:38:25 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-ai-meeting-helper-nodejs
+
+---
+# Meeting Helper with Azure Open AI
+
+This sample demonstrates generating action items and a meeting summary based on the attendees and transcription, then sending them to all participants.
+
+- To achieve this process, the user will schedule a meeting, which can be either a one-time or recurring event. Attendees interested in receiving the meeting summary and action items must individually subscribe to the meeting. For recurring meetings, multiple instances will be displayed, allowing users to subscribe to each instance separately.
+ 
+- After the meeting concludes, a webhook endpoint will be triggered in the background to extract the meeting transcription.
+
+- Using Azure OpenAI, the extracted meeting transcription will be processed to extract action items and a meeting summary. Finally, this information will be sent to all meeting attendees who have subscribed to the meeting.
+
+## Included Features
+* **Graph API:** Utilize graph APIs to retrieve meeting transcriptions and other meeting or user-related details.
+* **Azure Open AI:** Leverages Azure OpenAI for extracting action items and meeting summaries.
+* **Azure Table Storage:** Utilizes Azure Table Storage to store subscribed user information, including conversation IDs for sending meeting details.
+* **Self-Signed Certificate:** Uses Azure self-signed certificates for encryption and decryption to establish the meeting subscription.
+
+## Interaction with app
+
+ ![bot-ai-meeting-helperGif](Images/bot-ai-meeting-helperGif.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
```

Comment:
It is deployed app but referring any local certificate?

Reply:
no